ci/docs/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+#################################
+# RAPIDS Deployment Docs build script for CI #
+#################################
+
+if [ -z "$PROJECT_WORKSPACE" ]; then
+    echo ">>>> ERROR: Could not detect PROJECT_WORKSPACE in environment"
+    echo ">>>> WARNING: This script contains git commands meant for automated building, do not run locally"
+    exit 1
+fi
+
+export DOCS_WORKSPACE="$WORKSPACE"
```

Comment:
```suggestion
export DOCS_WORKSPACE="$WORKSPACE/docs"
```

this should be consistent with the `cudf` path below

- https://github.com/rapidsai/cudf/blob/288c81fd59d1b28541acb39235b314476f8a846a/ci/docs/build.sh#L13

Reply:
I had initially had it that way but removed it as I realized that there is no `docs` directory in the root directory of this repository. Am I missing something?

Reply:
Yes, the Jenkins job below clones the https://github.com/rapidsai/docs/ repo into the `$WORKSPACE/docs` directory.

- https://6abc.com/northern-lights-forecast-philadelphia-pennsylvania-nj/12137915/

Reply:
Oops, thanks.